components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -3127,22 +3128,14 @@ public static void updateExistingTenant(org.apache.stratos.common.beans.TenantIn
 
         // filling the first and last name values
         if (StringUtils.isBlank(tenantInfoBean.getFirstName())) {
-            try {
-                CommonUtil.validateName(tenantInfoBean.getFirstName(), "First Name");
-            } catch (Exception e) {
-                String msg = "Invalid first name is provided.";
-                log.error(msg, e);
-                throw new RestAPIException(msg, e);
-            }
+            String msg = "Invalid first name is provided.";
+            log.error(msg);
+            throw new RestAPIException(msg);
```

Comment:
It is not the practice to throw RestAPIException for specific errors. There are some other places we haven't fixed yet(not added from this PR), that we need to fix.